### PR TITLE
Code Review Request "Hello Microverse" Pull Request

### PR DESCRIPTION
**Pull request by dev-mamun**
It's a simple HTML & CSS project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (huseyinbiyik) [inline README.md]: - Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is FAQ. Please remove the template logo. Optionally, you can replace with your own logo if you hsve one. This is crucial to have a professional and instructive repository. 👌

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @dev-mamun,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Commits messages are meaningful 👌
- Linters setup properly and checks pass 🟢
- .gitignored file works very well 👌

## Required Changes ♻️
❗️ Please don't create a new branch, make all your changes on this branch.
- Please write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request as mentioned [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). You can change Pull Request description marked button below.
Screenshot 2023-01-10 at 8 36 13 PM

This is the PR Title↓ You can find the edit button on the right of the image below.
Screenshot 2023-01-10 at 8 39 30 PM

-As mentioned [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), please change your branch name to a meaningful one. For example: "project-setup".

> Respect the rules of the code review flow that your team follows. Use descriptive names for your branches (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars) and make sure that your 'feature branch' is up-to-date with the 'main branch' in your repository.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (dev-mamun) [reply README.md:1]: Could not change branch name because it will close this pull request.

**Review by dev-mamun — COMMENTED** [review]
remove template logo.

**Comment by dev-mamun** [comment]
> please change your branch name to a meaningful one.

Could not change branch name because it will close this pull request. @huseyinbiyik

**Comment by dev-mamun** [comment]
@huseyinbiyik any update?

**Comment by huseyinbiyik** [comment]
Hey @dev-mamun, I have just noticed your message. Sorry for this. I will get back to you very soon with an answer.

**Comment by huseyinbiyik** [comment]
Hey @dev-mamun, 
When you rename the branch name, it closes the PR immediately. You can rename the branch name and create a new Pull Request with the new branch name and include the old Pull Request link in the description of the new PR.

Also as I mentioned before, please write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request as mentioned [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). And make sure linter checks pass.

**Pull request merged**